final_task/rss_reader/custom_error.py

```diff
@@ -0,0 +1,58 @@
+"""Module with custom exceptions"""
+
+
+class Error(Exception):
+    """Base class for other exceptions"""
+
+    def __init__(self, message: str = None) -> None:
```

Comment:
A bit misleading typehint here:

You state that message should be `str` but it has default value of `NoneType`